src/components/Footer.css

```diff
@@ -20,12 +26,14 @@ footer .icon-container {
   font-size: 1.1em;
 }
 
-footer .icon .footer-icon {
-  max-width: 18px;
-  margin-right: 5px;
+@media all and (min-width: 767px) {
+  footer .icon .footer-icon {
+    max-width: 18px;
+    margin-right: 5px;
+  }
 }
 
-@media all and (max-width: var(--on-palm)) {
+@media all and (max-width: 767px) {
```

Comment:
good catch, that indeed didn't work!

As per spec: https://www.w3.org/TR/css-variables-1/#using-variables

> The var() function can be used in place of any part of a value in any property on an element. The var() function can not be used as property names, selectors, or anything else besides property values. (Doing so usually produces invalid syntax, or else a value whose meaning has no connection to the variable.)